src/effects/builtin/noisegen/noiseviewmodel.cpp

```diff
@@ -0,0 +1,64 @@
+/*
+ * Audacity: A Digital Audio Editor
+ */
+#include "noiseviewmodel.h"
+
+#include "global/translation.h"
+
+using namespace au::effects;
+
+QVariantList NoiseViewModel::types() const
+{
+    static QVariantList items;
+    if (items.empty()) {
+        items.append(QVariantMap {
+            { "text", QString::fromStdString(muse::trc("effects/noise", "White",
```

Comment:
we can use `qtrc` in this case 
see https://github.com/musescore/framework_tmp/blob/main/framework/global/translation.h